boxsdk/object/search.py

```diff
@@ -5,12 +5,147 @@
 from boxsdk.config import API
 from .base_endpoint import BaseEndpoint
 from boxsdk.util.translator import Translator
+import json
+
+class MetadataSearchFilter(object):
+    """
+    Helper class to encapsulate a single search filter. A search filter can only search against one template,
+    but can filter on many fields.
+    See :class:`MetadataSearchFilters`.
+    """
+    def __init__(self, template_key, scope):
```

Comment:
In the rest of the SDK, we default these to 'properties' and 'global'. I think we should do that here, as well.